www/src/components/plugin-searchbar-body.js

```diff
@@ -420,6 +423,21 @@ const Result = ({ hit, pathname, query }) => {
             fontSize: rhythm(4 / 7),
           }}
         >
+          {hit.repository &&
+            hit.repository.url.startsWith(
```

Comment:
Actually - one note here--startsWith [will break compatibility with IE11](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith), correct?

We can include a polyfill--but it'd probably be just as easy to do something like:

```js
hit.repository.url.indexOf('https://github.com/gatsbyjs/gatsby') === 0

// or

hit.repository.url.test(/^https:\/\/github.com/gatsbyjs/gatsby/)
```

neither are as succinct, but that's probably OK here!

Reply:
Just swapped startsWith with indexOf in both locations I do this check